dev/bots/test.dart

```diff
@@ -353,19 +332,18 @@ Future<void> _runBuildTests() async {
   final List<ShardRunner> tests = <ShardRunner>[
     for (final FileSystemEntity exampleDirectory in exampleDirectories)
         () => _runExampleProjectBuildTests(exampleDirectory),
-    if (branchName != 'beta' && branchName != 'stable')
-      ...<ShardRunner>[
-        // Web compilation tests.
-          () =>  _flutterBuildDart2js(
-          path.join('dev', 'integration_tests', 'web'),
-          path.join('lib', 'main.dart'),
-        ),
-        // Should not fail to compile with dart:io.
-          () =>  _flutterBuildDart2js(
-          path.join('dev', 'integration_tests', 'web_compile_tests'),
-          path.join('lib', 'dart_io_import.dart'),
-        ),
-      ],
+    ...<ShardRunner>[
+      // Web compilation tests.
+        () =>  _flutterBuildDart2js(
```

Comment:
nit: should this be indented back by 2 spaces?

Reply:
👍   Reformatted.